app/Services/Permissions/EntityPermission.php

```diff
@@ -331,4 +332,12 @@ protected function loadAllPermissions(User $user = null, Campaign $campaign = nu
         //dump('finished loading entities:');
         //dump($this->cachedEntityIds);
     }
+    public function resetPermissions(): void
```

Comment:
Missing empty line between old and new function, and function comment explaining what it does :)